src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -85,7 +85,8 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
                     style: {
                         shape: 'rect',
                         label: this._offerCredit ? 'credit' : undefined,
-                        ...pick(paypalOptions.style, 'layout', 'size', 'color', 'label', 'shape', 'tagline', 'fundingicons'),
+                        ...pick(paypalOptions.style, 'layout', 'size', 'color', 'label', 'shape', 'tagline', 'fundingicons', 'height'),
+                        height: Number(paypalOptions.style?.height),
```

Comment:
Do we need to do this? The pick method above should do this for us already

Reply:
We get string type value for height in paypalOptions but need number, so that is why i did that

Reply:
Then in that case you should be able to remove it height from pick statement.

Reply:
We changed type of height in bcapp so i removed  standalone height property